charts/datadog/values.yaml

```diff
@@ -803,10 +803,11 @@ agents:
       #
       create: false
 
-    ## @param securityContext - object - required
-    ## Provide securityContext configuration
+    ## @param seLinuxContext - object - required
+    ## Provide seLinuxContext configuration for PSP/SCC
     #
-    securityContext:
+    seLinuxContext:
```

Comment:
this is a breaking change. 
I think we need to style support the previous variable name, we can comment it and flag it as deprecated

Reply:
We do support it, though it's not documented anymore.